services/web/server/tests/unit/with_dbs/04/workspaces/test_workspaces__delete_workspace_with_content.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-statements
+
+
+from copy import deepcopy
+from http import HTTPStatus
+from unittest import mock
+
+import pytest
+from aiohttp.test_utils import TestClient
+from models_library.api_schemas_webserver.workspaces import WorkspaceGet
+from pytest_mock import MockerFixture
+from pytest_simcore.helpers.assert_checks import assert_status
+from pytest_simcore.helpers.webserver_login import UserInfoDict
+from pytest_simcore.helpers.webserver_projects import create_project
+from servicelib.aiohttp import status
+from simcore_service_webserver.db.models import UserRole
+from simcore_service_webserver.projects.models import ProjectDict
+
+
+@pytest.fixture
+def mock_storage_delete_data_folders(mocker: MockerFixture) -> mock.Mock:
+    mocker.patch(
+        "simcore_service_webserver.dynamic_scheduler.api.list_dynamic_services",
```

Comment:
For refactoring it is far easier if we patch the objects. So do
```python

import simcore_service_webserver.dynamic_scheduler.api

mocker.patch.object(
        simcore_service_webserver.dynamic_scheduler.api, "list_dynamic_services",
)

```

this way, when we rename the name of the modules, it helps to update the mocks. 

please do the same everywhere